SmartDeviceLink/SDLFileManager.m

```diff
@@ -260,6 +260,20 @@ - (void)uploadFiles:(NSArray<SDLFile *> *)files completionHandler:(nullable SDLF
     [self uploadFiles:files progressHandler:nil completionHandler:completionHandler];
 }
 
+- (void)uploadArtwork:(SDLArtwork *)artwork completionHandler:(nullable SDLFileManagerUploadArtworkCompletionHandler)completion {
+    if ([self.remoteFileNames containsObject:artwork.name] && !artwork.overwrite) {
```

Comment:
Is this necessary or can you just inspect the completion for the `cannotOverwriteError`? Simply to make sure that we aren't repeating the same code as exists in `uploadFile`.

Reply:
Fixed.